src/main/scala/org/allenai/pdffigures2/FindGraphicsRaster.scala

```diff
@@ -57,7 +59,8 @@ object FindGraphicsRaster {
     pixThreshold: Int,
     rescale: Int
   ): List[Box] = {
-    val pixelsToExplore = scala.collection.mutable.Stack[Int]()
+    val pixelsToExplore = mutable.Set[Int]()
```

Comment:
Doesn't each pixel have at most four neighbors? How could it get added on the stack more than four times?